stock_picking_batch_extended/views/stock_batch_picking.xml

```diff
@@ -170,8 +170,12 @@
         <field name="name">Batch Picking</field>
         <field name="res_model">stock.picking.batch</field>
         <field name="view_mode">tree,form</field>
-         <field name="view_id" ref="stock_batch_picking_tree"/>
-            <field name="search_view_id" ref="stock_batch_picking_search"/>
+        <field name="view_id" ref="stock_batch_picking_tree"/>
+        <field name="view_ids" eval="[
+            (5, 0, 0),
+            (0, 0, {'view_mode': 'tree', 'view_id': ref('stock_batch_picking_tree')}),
+            (0, 0, {'view_mode': 'form', 'view_id': ref('stock_batch_picking_form')})]"/>
```

Comment:
Good day @i-vyshnevska, we are porting this PR to 14.0. We are wondering what is the purpose of these three lines and are they needed for 14.0. Could you help to advise?

Reply:
@hailangvn yes this PR you also can include to migration
the first line is to clear previous relations if exist - so this will ensure there are no other views linked to this action
than we setting correct views we want to use

Reply:
Thanks @i-vyshnevska for the advise. We did tests with and without this code but could not find any difference. Could you please provide more background information that lead to this implementation?

Reply:
@hailangvn will fail if you try to update on installation, check for constrains "act_window_view_unique_mode_per_action" BTW could be different in v15